example/package.json

```diff
@@ -1,30 +1,23 @@
 {
   "name": "example",
   "version": "1.0.0",
-  "description": "An example of how to use Registry in TypeScript",
-  "main": "dist/example.js",
+  "description": "",
```

Comment:
Need to add a description

Reply:
I don't think so we don't publish this

Reply:
Additional info (e.g. description) always helps